medallion/test/test_memory_backend.py

```diff
@@ -372,3 +376,190 @@ def test_saving_data_file(self):  # just for the memory backend
             )
             objs = self.load_json_response(r_get.data)
             assert objs["objects"][0]["id"] == new_id
+
+    def test_get_collections_401(self):
+        r = self.client.get("/trustgroup1/collections/")
+        self.assertEqual(r.status_code, 401)
+
+    def test_get_collections_403(self):
```

Comment:
We don't need to define actual tests for these. Just a comment that Medallion will never return a 403 for this endpoint is fine.

Reply:
Noted.